UnityGLTF/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs

```diff
@@ -0,0 +1,388 @@
+#if UNITY_2017_1_OR_NEWER
```

Comment:
Just curious, but what's specific to 2017.1?

Reply:
`UnityEditor.Experimental.AssetImporters` (scripted AssetImporters) was introduced in 2017.1, I believe (unless it's in a 5.6 patch)